src/index.html

```diff
@@ -4,9 +4,304 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;1,100&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+    crossorigin="#">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo-link" href="#">
+        <img class="header__logo" src="images/logo.png"
+        alt="logo">
+      </a>
+
+      <nav class="header__nav">
+        <ul class="header__list">
+          <li class="header__item">
+            <a class="header__nav-item is-active" href="#">apple</a>
+          </li>
+
+          <li class="header__item">
+            <a class="header__nav-item" href="#">samsung</a>
+          </li>
+
+          <li class="header__item">
+            <a class="header__nav-item" href="#">smartphones</a>
+          </li>
+
+          <li class="header__item">
+            <a class="header__nav-item" data-qa="nav-hover"
+            href="#">laptops & computers</a>
```

Comment:
Fix this issue in all code.
<img width="1001" alt="image" src="https://github.com/mate-academy/layout_catalog/assets/54860641/83870086-f453-4a87-9f39-b269baa0fe00">